LWCE_Core/Src/XComLongWarCommunityEdition/Classes/LWCETacticalVisibilityHelper.uc

```diff
@@ -450,12 +450,11 @@ protected function SetVisibilityMarkers(XGUnit kActiveUnit, XGUnit kHelper)
         }
 
         bFlanked = false;
-
         if (bVisible)
         {
-            // For some reason, after loading a save, IsFlankedBy stops working for our helper units.
-            // Flank indicators are disabled entirely until we can figure that out.
-            // bFlanked = kUnit.CanUseCover() && (!kUnit.IsInCover() || kUnit.IsFlankedBy(kHelper));
+            // Condition from UISightlineHUD_SightlineContainer
+            bFlanked = !kUnit.IsFlying() && kUnit.CanUseCover()
+                && ( kUnit.IsFlankedByLoc(kHelper.Location) || kUnit.IsFlankedBy(kHelper) || !kUnit.IsInCover() );
```

Comment:
Very minor optimization, but I had `!kUnit.IsInCover()` as the first condition because it's a very cheap check, whereas I don't know how expensive the other two are.